### PR TITLE
Update 0.119 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ General Changes
 * Deprecated ``task.http-notification-threads`` config option in favor of two
   independent options: ``task.http-response-threads`` and ``task.http-timeout-threads``.
 * Improve handling of negated expressions in join criteria.
-* Add support for Cassandra BLOB types.
+* Support ``ADD COLUMN`` and ``RENAME COLUMN`` in :doc:`/sql/alter-table`.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: wan't rename column already in Presto?

File: presto-docs/src/main/sphinx/release/release-0.119.rst [hunk 2]
```diff
@@ -43,18 +43,23 @@ CLI Changes
...
+Cassandra Changes
+-----------------
+
+* Add support for Cassandra blob types.
+
 Hive Changes
 ------------
 
-* Add ``ALTER TABLE ADD COLUMN``
-* Add ``ALTER TABLE RENAME COLUMN``
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: This is the important part.  It now works with Hive.

**Comment by dain** [comment]
Looks good

File: presto-docs/src/main/sphinx/release/release-0.119.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: You're right, it does

File: presto-docs/src/main/sphinx/release/release-0.119.rst (hunk 2 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.119.rst]: I updated the note to include `ALTER TABLE`

**Pull request merged**